### ISSUE TITLE
[BUG]: God Hand SLUS-21503: Outline around minimap

### ISSUE THREAD

**Issue by mbriar** [PCSX2/pcsx2#8137]
### Describe the Bug

In God Hand, there is an outline around the minimap UI element (see screenshots below), it occurs in vulkan, opengl and software renderers. However, It goes away if you *enable* 'GPU Palette Conversion'.

### Reproduction Steps

Start a new game, skip the intro cutscene, the minimap is immediately visible.

### Expected Behavior

No outline around the minimap, as shown in the screenshot below with 'GPU Palette Conversion' enabled.

### PCSX2 Revision

v1.7.4075

### Operating System

Linux (64bit) - Specify distro below

### If Linux - Specify Distro

Arch

### CPU

Ryzen 3700x

### GPU

RX 6700 XT

### GS Settings

All default.

### Emulation Settings

All Default.

### GS Window Screenshots

How it looks with default settings on vulkan/opengl/software:

![God Hand_SLUS-21503_20230213124609](https://user-images.githubusercontent.com/114156793/218452598-74ba5bbe-96ef-4efc-ab20-586d79206fad.png)

How it looks with 'GPU Palette Conversion' enabled:

![God Hand_SLUS-21503_20230213124742](https://user-images.githubusercontent.com/114156793/218452733-1a1d7452-5e9a-4985-8ac6-5596c61234eb.png)

### Logs & Dumps

GSdump: 
[God Hand_SLUS-21503_20230213124609.zip](https://github.com/PCSX2/pcsx2/files/10721957/God.Hand_SLUS-21503_20230213124609.zip)

**Comment by Mrlinkwii** [comment]
can this be checked  on latest nightly

**Comment by mbriar** [comment]
![Screenshot_20240629_232240](https://github.com/PCSX2/pcsx2/assets/114156793/639683e5-5c4d-440f-8663-f54c8735f6e4)

Still looks like this on v1.7.5947. It also seems like the 'GPU Palette Conversion' option is gone now?

**Comment by JordanTheToaster** [comment]
> Still looks like this on v1.7.5947. It also seems like the 'GPU Palette Conversion' option is gone now?

Palette conversion doesn't fix it and it's there in software on the dump you posted.

**Comment by mbriar** [comment]
It must have fixed it at some point or I wouldn't have gotten the screenshot without the issue. But yes, software never helped, neither with a dump or the actual game.

**Comment by Mrlinkwii** [comment]
is this still an issue ?

**Comment by birthtothunder** [comment]
Seems to be fixed as of 2.3.168

![Image](https://github.com/user-attachments/assets/6073971c-ce51-4c6a-af6e-b9374b27fd2e)

**Comment by birthtothunder** [comment]
Seems to still be an issue as of latest update (2.3.232)
![Image](https://github.com/user-attachments/assets/ab210870-a5e1-47b6-b414-3f11614c0f77)

**Comment by refractionpcsx2** [comment]
Thanks for confirming with real hardware, sanity checks are always good :)

**Comment by Ir-boop** [comment]
Yeah, no problem. It's somewhat hard to notice on tv screen so not surprising if so many people not notice it yet.
